src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CompilerExtensions.projitems

```diff
@@ -556,6 +555,10 @@
     <Compile Include="$(MSBuildThisFileDirectory)Diagnostics\DiagnosticAnalyzerCategory.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Diagnostics\DiagnosticCategory.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Diagnostics\IBuiltInAnalyzer.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Diagnostics\IPragmaSuppressionsAnalyzer.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)..\..\..\..\Compilers\Core\Portable\DiagnosticAnalyzer\AnalyzerConfigOptionsDictionary.cs">
+      <Link>Diagnostics\AnalyzerConfigOptionsDictionary.cs</Link>
+    </Compile>
```

Comment:
Can we file something into the public API backlog to look at making this public? I realize this shared project already includes many compiler files, but a quick glance at them implies most of them are just helpful extensions over framework methods. If we're actually needing compiler types in this case, seems we should just make them public.

Reply:
(I'm OK with doing the review separately, but want to be cognizant of the debt we're creating.)

Reply:
We don't really *need* the type. It's just convenient to share it just so that we don't have two physical copies. Same as other internal utilities.
I'm not sure if there is enough value in this type to it a public API at this point.